ls1intum/Artemis
# `Assessment`: Show logic for assessment training

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

[ ] * - not needed

### Motivation and Context


In order to add clarity to the tutors on how exactly their assessment is evaluated, a new section in the instructions view is added. This section should only be shown when tutor is assessing example submissions.
Extra heading is also added in example submissions screen (as shown in the screenshot below)

### Description

I added a new section in the `assessment-instructions.component` as well as the new heading in the `example-text-submissions` and exaple-modeling-submissions` components.

### Steps for Testing

Prerequisites:
- 1 Tutor

1. Log in to Artemis
2. Navigate to Course Management
3. Navigate to Assessment Dashboard of a Text or Modelling exercise that has Example Submissions to **assess** or create example submissions yourself. (you need to be an instructor to create example submissions) 
4. Click on Assess example submission.
5. In the instructions panel on the right you should see the "Assessment Training Evaluation Criteria" section as well as the header "This is an Assessment Training". (see screenshots)

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots


#### New header indicating that it's an assessment training
<img width="1297" alt="Screen Shot 2021-11-27 at 15 53 35" src="https://user-images.githubusercontent.com/51077603/143686368-9db35556-b34a-423f-8c1d-3182229c5ce5.png">

#### New section explaining the assessment logic
<img width="760" alt="logic" src="https://user-images.githubusercontent.com/51077603/143772102-e5699e76-bd91-4179-90aa-a5875b190c22.png">
